Jetchat/app/src/main/java/com/example/compose/jetchat/conversation/UserInput.kt

```diff
@@ -219,7 +223,12 @@ private fun SelectorExpanded(
 @OptIn(ExperimentalAnimationApi::class)
 @Composable
 fun FunctionalityNotAvailablePanel() {
-    AnimatedVisibility(visible = true, enter = fadeIn()) {
+    AnimatedVisibility(
+        visibleState = remember { MutableTransitionState(false).apply { targetState = true } },
+        // Remove if https://issuetracker.google.com/190816173 is fixed
```

Comment:
BTW, the default (after fixing the issue linked above) will be expanding and shrinking diagonally while fading, unless the AnimatedVisibility is in a RowScope/ColumnScope.  So you'll need these customizations on enter/exit anyways if you prefer horizontal expanding and shrinking.  :-)

Reply:
ACK!